src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String clr = colorSupplier.getRandomColor();
+        int index = new Random().nextInt(100);
```

Comment:
Think about which variables should be local in the method and which should be class-level. `new Random()` and new `ColorSupplier()` object will be created on each `getRandomBall() `call. How can you avoid this?